script.js

```diff
@@ -0,0 +1,59 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const bookTitle = document.getElementById('title');
+  const bookAuthor = document.getElementById('author');
+  const article = document.getElementById('bookList');
+
+  const addButton = document.getElementById('add');
+  const saveItemToLocalStorage = (title, author) => {
+    const savedBooks = JSON.parse(localStorage.getItem('savedBooks')) || [];
+    savedBooks.push({ title, author });
```

Comment:
Well done on your implementation! However, I'd like to suggest a slight improvement that could enhance your code quality.
- [ ] [Optional]  Currently, you are using the `array.push()` method to add new elements to an array. While this approach works well, I recommend considering the use of the `array.concat()` method instead. The [concat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) method of [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) instances allows you to merge two or more arrays. The key advantage here is that it doesn't modify the existing arrays; instead, it returns a new array that combines the contents of the original arrays. This can help maintain data [immutability](https://developer.mozilla.org/en-US/docs/Glossary/Immutable) and prevent unexpected side effects.

By using `array.concat()` in your context, you'll be creating a new array that includes your newly generated book while keeping the original array intact. This can contribute to a more robust and maintainable codebase. Keep up the excellent work, and happy coding! 🌟📝🚀"